lab_3_keywords_textrank/main.py

```diff
@@ -233,7 +273,18 @@ def add_edge(self, vertex1: int, vertex2: int) -> int:
                 0 if edge was added successfully, otherwise -1
         In case of vertex1 being equal to vertex2, -1 is returned as loops are prohibited
         """
-        pass
+        if vertex1 == vertex2:
+            return -1
+        for one_vert in vertex1, vertex2:
```

Comment:
can you move it to a help method?

Reply:
what do you mean? putting the check "whether the vertexes are the same" into another method?

Reply:
the body of this loop. and then just call it twice

Reply:
so i should write one more method in this class, which will be adding numbers to a matrix, and then call this method in def add edge?